serviceCode=U5G
rbu=rbu2
customer=u5g
bo.certificates.classifier=mutu
keystore.file=${acs.mutu.keystore}
keystore.password=${acs.mutu.keystore.password}
#ve.server.url=https://ssl-qlf-u5g-fo-acs-ve.wlp-acs.com

# HUB configuration
hub.customer=h5g
hub.keystore.file=${hub.mutu.keystore}
hub.keystore.password=${hub.mutu.keystore.password}
hub.referential.service.url=https://ssl-qlf-h5g-fo-hub-cr.ita.stg.as8677.net/referential/rest
hub.referential.server.url=https://ssl-qlf-h5g-fo-hub-cr.ita.stg.as8677.net
hub.rba.service.url=https://ssl-qlf-h0g-fo-hub-rba.ita.stg.as8677.net/rba/rest

# BO DS
datasources.mystique.pool-name=acs3_bo
datasources.mystique.jndi-name=java:jboss/datasources/mystique
datasources.mystique.url=jdbc:mysql://acs-mysql:3306/U5G_ACS_BO?autoReconnect=true&amp;cacheResultSetMetadata=true&amp;useUnicode=true&amp;characterEncoding=utf-8
datasources.mystique.user=root
datasources.mystique.password=root
datasources.mystique.driver=mysql
datasources.mystique.validation.valid-connection-checker.class-name==org.jboss.jca.adapters.jdbc.extensions.mysql.MySQLValidConnectionChecker
datasources.mystique.validation.exception-sorter.class-name==org.jboss.jca.adapters.jdbc.extensions.mysql.MySQLExceptionSorter
